esmf_regrid/schemes.py

```diff
@@ -1555,13 +1722,21 @@ def __init__(
         tgt_location : str or None, default=None
             Either "face" or "node". Describes the location for data on the mesh
             if the target is not a :class:`~iris.cube.Cube`.
+        extrapolate_gaps : bool, default=False
+            Use a standard set of ESMF arguments for extrapolation which achieves
+            continuity with bilinear regridding. Useful for situations where gaps
+            between cells would be masked.
+        esmf_args : dict, optional
+            A dictionary of arguments to pass to ESMF.
 
         Raises
         ------
         ValueError
             If ``use_src_mask`` or ``use_tgt_mask`` are True while the masks on ``src``
             or ``tgt`` respectively are not constant over non-horizontal dimensions.
         """
+        if extrapolate_gaps:
+            esmf_args = STANDAR_BILINEAR_EXTAP_ARGS
```

Comment:
I guess same applies here as above : an  'update' may be more useful than an overwrite